sewer/dns_providers/route53.py

```diff
@@ -0,0 +1,95 @@
+import collections
+import boto3
+
+from . import common
+
+
+# most code of this class is copy from certbot's route53 dns plugin.
+class Route53Dns(common.BaseDns):
+    ttl = 10
+
+    def __init__(self, access_key_id=None, secret_access_key=None):
+        if access_key_id and secret_access_key:
+            # use user given credential
+            self.r53 = boto3.client(
+                "route53", aws_access_key_id=access_key_id, aws_secret_access_key=secret_access_key,
+            )
```

Comment:
`sewer` tries to have timeouts for network calls so that users do not end up waiting forever; 
https://github.com/komuw/sewer/blob/02ab3382f0f6f4a73414734bae34d9f67da90254/sewer/dns_providers/cloudflare.py#L24   
https://github.com/komuw/sewer/blob/02ab3382f0f6f4a73414734bae34d9f67da90254/sewer/dns_providers/cloudflare.py#L36  

could you do the same for boto3?
```python
from botocore.client import Config

aws_config = Config(connect_timeout=30.0, read_timeout=30.0)
self.r53 = boto3.client("route53", config=aws_config)
```

Reply:
Added.